pkg/config/interface.go

```diff
@@ -44,6 +44,9 @@ type ProjectConfig interface {
 	SendFlagDecisions() bool
 	GetSdkKey() string
 	GetEnvironmentKey() string
+	GetAttributes() []entities.Attribute
+	GetEvents() []entities.Event
+	GetOptimizelyAudiences() []entities.OptimizelyAudience
```

Comment:
We can change to GetAudiences() here and build OptimizelyAudiences in OptimizelyConfig.